specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -337,6 +337,270 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/proximityPlacementGroups/{proximityPlacementGroupName}": {
+      "put": {
+        "tags": [
+          "ProximityPlacementGroups"
+        ],
+        "operationId": "ProximityPlacementGroups_CreateOrUpdate",
```

Comment:
should this be marked as a long-running operation  "x-ms-long-running-operation": true ?

Reply:
@aspand

Reply:
I just copied from what we have in 2018-10-01 version.

Reply:
No ProximityPlacementGroups operation is done in sync path aleady discuused this with Anu in https://github.com/Azure/azure-rest-api-specs/pull/5507